modules/message/sheet/preview.js

```diff
@@ -21,6 +21,24 @@ exports.create = function (api) {
       var isPrivate = msg.value.private
       var isRoot = !msg.value.content.root
       var recps = (msg.value.content.recps || []).filter(id => id !== msg.value.author)
+
+      // handle too many private recipients
+      if (isPrivate && recps.length > 7) {
+        return {
+          content: [
+            h('h2', ['Too many recipients']),
```

Comment:
These strings should use `i18n` to make them localisable. It should just be a matter of wrapping with a call to `i18n`, for example:

```
h('h2', [i18n('Too many recipients')])
```

But you'll need to make sure i18n is included in scope (I think it is in this file).